### PR TITLE
Included time zone considerations and tips, moved the parking lot def…

### PR DESCRIPTION
**Pull request by madossan01**
Included time zone considerations and tips, moved the parking lot definition to a note block and added the remote working section.

### CODE REVIEW

File: agile-development/stand-ups/readme.md [hunk 1]
```diff
@@ -8,9 +8,12 @@ The stand-up is a time-boxed ceremony that is held each day of the sprint. In th
 
 During the stand-up, additional discussions may arise. Make sure that someone adds them to the parking lot for after meeting discussion.  
 
-After that point, the stand-up is concluded. The items in the parking lot may take place right after. However, participation in the parking lot discussion is optional for all members except those explicitly needed for discussion of the issues raised.
+After that point, the stand-up is concluded.
 
-The term parking lot refers to a bucket of comments, concerns, or questions that will be discussed and/or addressed at a later point with potentially fewer contributors. This is part of a strategy to avoid letting the discussion in a meeting shift to a subject that is not aligned with the meeting goals and/or decisions.
+As a good practice, the items in the parking lot take place right after the stand-up. However, you can opt to discuss the parking lot items another time before the next stand-up.
```
Comment (meulta) [inline agile-development/stand-ups/readme.md:13]: nit: missing "at": "discuss the parking lot items at another time"

**Review by meulta — APPROVED** [review]
LGTM with a comment

**Review by fnocera — APPROVED** [review]
Great clarifying edits and additions.

**Comment by fnocera** [comment]
@madossan01 please address the comments then it should be good to complete.

**Review by davidkpiano — APPROVED** [review]
Also might be good to link here: https://www.scrum.org/resources/blog/daily-scrum-tips-tactics#:~:text=Use%20a%20parking%20lot.

**Pull request merged**